inst/include/Rcpp/exceptions.h

```diff
@@ -126,40 +126,49 @@ namespace Rcpp{
 
 } // namespace Rcpp
 
-inline SEXP nth(SEXP s, int n) {
-  return Rf_length(s) > n ? (n == 0 ? CAR(s) : CAR(Rf_nthcdr(s, n))) : R_NilValue;
+namespace internal {
```

Comment:
Looks like this accidentally was left outside of the outer  `namespace Rcpp` bombing compilation:

https://travis-ci.org/RcppCore/Rcpp#L2618

```
na.cpp:27:12: error: reference to ‘internal’ is ambiguous
     return internal::Rcpp_IsNA(x);
            ^
In file included from /home/travis/build/RcppCore/Rcpp/Rcpp.Rcheck/Rcpp/include/RcppCommon.h:122:0,
                 from /home/travis/build/RcppCore/Rcpp/Rcpp.Rcheck/Rcpp/include/Rcpp.h:27,
                 from na.cpp:22:
/home/travis/build/RcppCore/Rcpp/Rcpp.Rcheck/Rcpp/include/Rcpp/exceptions.h:129:20: note: candidates are: namespace internal { }
 namespace internal {
                    ^
In file included from /home/travis/build/RcppCore/Rcpp/Rcpp.Rcheck/Rcpp/include/Rcpp.h:27:0,
                 from na.cpp:22:
/home/travis/build/RcppCore/Rcpp/Rcpp.Rcheck/Rcpp/include/RcppCommon.h:45:24: note:                 namespace Rcpp::internal { }
     namespace internal {
                        ^
```